params/ukf.yaml

```diff
@@ -38,6 +38,15 @@ ukf_filter_node:
 # Defaults to "robot_localization_debug.txt" if unspecified. Please specify the full path.
         debug_out_file: /path/to/debug/file.txt
 
+# Whether we'll allow old measurements to cause a re-publication of the updated state
+        permit_corrected_publication: false
```

Comment:
I don't see any parameter gets to set this value to use the parameterization. How is this being "gotten" by the system?

Reply:
Whoops, good catch.